controller/work_item_link_type.go

```diff
@@ -222,6 +223,26 @@ func (c *WorkItemLinkTypeController) List(ctx *app.ListWorkItemLinkTypeContext)
 	})
 }
 
+// ListTypeCombinations runs the list-type-combinations action.
+func (c *WorkItemLinkTypeController) ListTypeCombinations(ctx *app.ListTypeCombinationsWorkItemLinkTypeContext) error {
+	return application.Transactional(c.db, func(appl application.Application) error {
+		m, err := appl.WorkItemLinkTypeCombinations().List(ctx, ctx.WiltID)
```

Comment:
unless the work item link type is already specific to a space? I.e., if each space redefines the work item link types, then there would be no need to specify the `spaceID` in the query.